extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js

```diff
@@ -67,6 +67,49 @@ describes.realWin('amp-jwplayer', {
           'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html');
     });
   });
+  it('renders with a playlist and all parameters', () => {
+    return getjwplayer({
+      'data-playlist-id': '482jsTAr',
+      'data-player-id': 'sDZEo0ea',
+      'data-content-search': 'dog',
+      'data-content-contextual': true,
+      'data-content-recency': '9D',
+      'data-content-backfill': true,
+    }).then(jw => {
+      const iframe = jw.querySelector('iframe');
+      expect(iframe).to.not.be.null;
+      expect(iframe.tagName).to.equal('IFRAME');
+      expect(iframe.src).to.equal(
+          'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=dog&contextual=true&recency=9D&backfill=true');
```

Comment:
Wouldn't this embed not be contextual (search=dog)?  How is `"/players/*.html"` expected to handle these parameters? `"/players/*.js"` Does not get passed any of these parameters, so the player config is not affected at all.

It's seems like you should just always pass the AMP embed whatever page info a contextual embed would need because you don't know how the player is going to be configured, and the player config can change without the AMP embed being changed.